pkg/services/thumbs/dashboardthumbsimpl/xorm_store.go

```diff
@@ -15,17 +15,17 @@ type xormStore struct {
 	db db.DB
 }
 
-func (ss *xormStore) Get(ctx context.Context, query *thumbs.GetDashboardThumbnailCommand) (*thumbs.DashboardThumbnail, error) {
-	err := ss.db.WithDbSession(ctx, func(sess *db.Session) error {
-		result, err := findThumbnailByMeta(sess, query.DashboardThumbnailMeta)
+func (ss *xormStore) Get(ctx context.Context, query *thumbs.GetDashboardThumbnailCommand) (result *thumbs.DashboardThumbnail, err error) {
+	err = ss.db.WithDbSession(ctx, func(sess *db.Session) error {
+		thumb, err := findThumbnailByMeta(sess, query.DashboardThumbnailMeta)
```

Comment:
nit: could we simplify this by setting this to result directly?

Reply:
no strong opinions but since we return the `result` I think it's better to set it after checking the error
my personal preference though would be: not to use named return variables at all (I find the code that way less straightforward)

Reply:
no strong opinions but since we return the `result` I think it's better to set it after checking the error
my personal preference though would be: not to use named return variables at all (I find the code that way less straightforward)